src/simplify_algebra.cpp

```diff
@@ -1200,6 +1200,9 @@ struct find_div_const
         auto ins   = r.result;
         auto c_ins = r.instructions["c"];
 
+        if(shape::is_integral(ins->get_shape().type()))
+            return;
```

Comment:
i don't get why it would skip on integral types ? `find_div_const` should work for all types.

Reply:
Because the `1/x`(with `x > 0`) is always 0 for integral types. So its not a valid transformation.